### ISSUE TITLE
[Google Search API] information box on the right is not scraped on google.fr

### ISSUE THREAD

**Issue by sonika-serpapi** [serpapi/public-roadmap#2030]
A user reached out that the information box on the right is not scraped on google.fr. 
This looks a bit different than the typical knowledge graph, as it includes some search results as well.

I was unable to recreate this my end, but the user attached this screenshot:

![image (3)](https://github.com/user-attachments/assets/4f353d7b-5e77-4fb0-91f6-397d447f7634)

Our playground seems to return a different layout for this query:

<img width="1345" alt="Screenshot 2024-10-14 at 10 37 54 AM" src="https://github.com/user-attachments/assets/423fa1c9-84ec-4662-a1dc-e334877c97cc">

Public links: [Playground 1](https://serpapi.com/playground?q=expert+comptable+narbonne&location=France&google_domain=google.fr&gl=fr&hl=fr&device=desktop&google_domain=google.fr&newPara=chips+as_sitesearch+highlight+stick+lr+as_qdr+votes+site+safe+google_domain+si+form+filter+as_epq+client+ie+oe+pws+ibp+source+rlz+ijn)

Internal links: [Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000071167?view=List) | [Search Inspect 1](https://serpapi.com/searches/670d4e012da157daf4f56ac8/inspect)

**Comment by marm123** [comment]
I'm not sure if this can be scraped within the same API call, but you can get these results by searching for the business, as they are just `organic_results` from that query:

<img width="1670" alt="image" src="https://github.com/user-attachments/assets/0ba9ba16-fd18-4d25-85de-1888e77af722">

If this can not be scraped in a single API call, this might be a valid workaround for a user.

**Comment by schaferyan** [comment]
@sonika-serpapi by `place_id` do you mean `ludocid`? The `ludocid` and `lsig` parameters currently don't work with SerpApi, as documented [here](https://github.com/serpapi/public-roadmap/issues/1175).

**Comment by sonika-serpapi** [comment]
Yes, I meant `ludocid` in this case. Thank you for linking that issue @schaferyan. This makes sense why the results do not show up.
